modules/apps/object/object-test/src/testFunctional/tests/Object.testcase

```diff
@@ -21,6 +21,143 @@ definition {
 		}
 	}
 
+	@description = "LPS-TEST - Verify if it is possible to create a Custom Object"
+	@priority = 5
+	test CanCreateCustomObject {
+		ObjectAdmin.addObjectViaAPI(
+			labelName = "Custom Object 01",
+			objectName = "CustomObject01",
+			pluralLabelName = "Custom Objects 01");
+
+		ObjectAdmin.openObjectAdmin();
+
+		Refresh();
+
+		Pause(locator1 = 3000);
```

Comment:
Can you explain to me the reason for using "refresh" and "pause"? As you had added an object via the API, it would already be there when you went to the objects page. In this case, it is not necessary to use these functions.

Reply:
But this is just an observation, as the focus of this test is using a macro through the UI to add the object.

Reply:
The reason for using refresh and pause was kind of a validation and for pause, the better visualization of the test in particular. But, yesterday I attended a class that addressed this issue of the pause, and how much it influences the performance of the C.I.
So they recommended some alternative functions, in case some waiting is really necessary in the test, because the client won't want to wait "x" seconds to click on such functionality, you know? Appeared, it will click, so they recommended: 
- WaitForElementPresent
- WaitForElementNotPresent
- while (IsElementNotPresent(locator1 = "Staging#PROCESS_RESULT")) {
Pause(locator1 = "1000");

With these macros the performance of the C.I will improve a lot, compared to just the pause.

Reply:
And these functions mentioned are very useful and, as you said, lead to an improvement in the performance of the ci